includes/blocks/course-theme/class-lesson-module.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * File containing the Module_Title class.
```

Comment:
```suggestion
 * File containing the Lesson_Module class.
```

Reply:
🤦 Missed those parts. Updated the names in all places. 👍

Reply:
Hey Nurguly! Sorry that I didn't notice in the same review, but in a final check to approve, I noticed the same thing to the "lesson title" / "post title". I think we have to fix it in some parts that are still referencing it as lesson title. :)